src/sentry/integrations/github/client.py

```diff
@@ -88,7 +89,7 @@ def get_tree(self, repo_full_name: str, tree_sha: str) -> List[str]:
                 logger.warning(
                     f"The tree for {repo_full_name} has been truncated. Use different a approach for retrieving contents of tree."
                 )
-            tree = trim_tree(contents["tree"], ["python"])
```

Comment:
We move the responsibility of flattening the tree to a list of files to `get_repo_files`.